### ISSUE TITLE
submit for fwitter when I'm already on a team shows spinner forever

### ISSUE THREAD

**Issue by jonopens** [learn-co-curriculum/sinatra-fwitter-group-project#31]
I am on a team with David Squires for the Fwitter exercise. In the last 10 minutes or so, he submitted with me as his teammate, but it didn't show me as having submitted, so I thought I had to also. 
I had forked the lab from him (/stopvoldemort) and we worked in parallel, so when we completed it, we merged into master, and then I updated my local.

Anyway, when I try to submit by myself, it says I can't because my fork is the same as the lesson's (which I can confirm is not true just by `cat app/controllers/application_controller.rb` in terminal from my fwitter directory). So then I noticed I didn't have David on my team, it said I was solo, so I tried to add him. When I tried to submit that team add request, I get a spinner and an error in the network traffic that says I'm already part of a project. So my issue is that I should already be all green if I'm already part of a team that successfully submitted and not have ability to submit again. When I refresh the page, it doesn't tell me that I'm already "done" with the lab.

![screen shot 2017-09-21 at 1 22 16 pm](https://user-images.githubusercontent.com/10333387/30709797-6e8c0b5a-9ed1-11e7-8c02-1c2b72f4b28e.png)
![screen shot 2017-09-21 at 1 22 40 pm](https://user-images.githubusercontent.com/10333387/30709798-6e8e54f0-9ed1-11e7-9b6a-73881250c98a.png)

**Comment by curiositypaths** [comment]
Hi there,

[Raising issues on GitHub](http://help.learn.co/improving-learn/contributing-to-lessons/raising-an-issue-with-a-lab-or-lesson) should be reserved for problems with the curriculum — not difficulties solving a lab, mistakes fixed via [a quick edit](http://help.learn.co/improving-learn/contributing-to-lessons/can-i-edit-a-lesson), or bugs with the Learn platform (such as [lights not turning green](http://help.learn.co/your-learn-account/learn-lights/lights-not-turning-green) or [IDE issues](http://help.learn.co/the-learn-ide)).

This issue would be perfect for the [Ask a Question](http://help.learn.co/ask-a-question/where-can-i-ask-a-question-about-a-lesson) (AAQ) feature, where the [Technical Coaches](http://help.learn.co/instructional-support/receiving-course-support/who-are-the-technical-coaches) (on [supported tracks](http://help.learn.co/instructional-support/receiving-course-support/does-my-program-or-course-have-technical-coach-support)) and fellow community members can help you figure it out. For some tips on how to take full advantage of AAQ, check out [this helpful article](http://help.learn.co/ask-a-question/how-to-optimize-time-in-ask-a-question) in the [Learn Help Center](http://help.learn.co/).

If, after consulting the Help Center and AAQ, you still feel that this is an issue with the curriculum itself, please reopen the issue and provide more detail about why it's a curriculum issue, the steps needed to reproduce the issue, and any additional relevant information about your development environment (local or IDE).

Thanks! 💙
